teerex/Cargo.toml

```diff
@@ -12,8 +12,9 @@ edition = "2021"
 codec = { version = "3.0.0", default-features = false, features = ["derive"], package = "parity-scale-codec" }
 hex = { default-features = false, version = "0.4.3", features = ["alloc"] }
 log = { version = "0.4.14", default-features = false }
-scale-info = { version = "2.0.1", default-features = false, features = ["derive"] }
+scale-info = { version = "2.10.0", default-features = false, features = ["derive"] }
 serde = { version = "1.0", features = ["derive"], optional = true }
+webpki = { git = "https://github.com/rustls/webpki", version = "=0.102.0-alpha.3", rev = "da923ed", package = "rustls-webpki", default-features = false, features = ["alloc", "ring"] }
```

Comment:
We did not have a code change in this pallet, why do we have a new dependency here?

Reply:
We do have [changes](https://github.com/integritee-network/pallets/pull/228/files#diff-b00e78f3cf2794df255a5ca3a7682559534cddf0c0986aff24d680e051abc422) in `teerex/src/lib.rs`

Reply:
Ah yes. The git diff was not really clear, all good!